deploy/kubernetes/dolphinscheduler/values.yaml

```
@@ -583,6 +583,38 @@ worker:
   # -- Update strategy
   # ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   #
+  # -- Direct Secret Mounting
+  # Mount secrets directly as environment variables
+  # Single secret
+  envFromSecret: "git-sync-secret"
+  
+  # Multiple secrets (advanced use cases)
+  # Use when you have variables distributed across different secrets
+  # envFromSecrets:
+  #   - "git-sync-secret"        # Git synchronization credentials
+  #   - "database-secret"        # Database connection details
+  #   - "external-api-secret"    # Third-party API keys
+  
+  # -- Init Container for Advanced Processing
+  # Use when you need to transform, validate, or prepare configuration files
+  initContainers:
```

I think `initContainers` should be disable as default.     
```suggestion
  initContainers: {}
```

The following examples can be retained, but they should be treated as comments.